doc/source/language/delegates.rst

```diff
@@ -402,5 +403,5 @@ can't be changed easily.*)
     print "altpos at altitude 1000 is " + altpos:call(1000).
 
   However, if you like you can make your own user function that is a
-  wrapper around a structure suffix call, and make a delegate of THAT.
+  wrapper around a structure suffix call, and make a delegate of *that*.
```

Comment:
+1 to bold in place of italics.  I have no idea if there is any kind of a standard, but I treat bold as emphasis, and italics as a change in tone or other subtle variation that comes out in spoken but not written language.